sphinx/environment/__init__.py

```diff
@@ -430,7 +430,7 @@ def find_files(self, config, buildername):
         # is set for the doc source and the mo file, it is processed again from
         # the reading phase when mo is updated. In the future, we would like to
         # move i18n process into the writing phase, and remove these lines.
-        if buildername != 'gettext':
+        if builder.use_message_catalog:
```

Comment:
Essentially, this process is not related "find files". So it would be nice to move to other process. I'll think it later.